packages/apps/spaces/app/organization/[id]/src/components/Tabs/panels/AboutPanel/branches/ParentOrgInput.tsx

```diff
@@ -42,12 +42,16 @@ export const ParentOrgInput: React.FC<ParentOrgInputProps> = ({
     },
     sort: undefined,
     where: {
-      filter: {
-        property: 'ORGANIZATION',
-        value: searchTerm,
-        operation: ComparisonOperator.Contains,
-        caseSensitive: false,
-      },
+      AND: [
```

Comment:
this change looks suspicious, and condition requires more than 1 filter

Reply:
the backend accepts this with no problem. sending the filter as it was initiallt does not actually return the value even if it's in db. Doing it in this way it works.

Reply:
It's also how we're doing it in the finder table with the search.